configs/sst_kernel_tps-tracing.yaml

```diff
@@ -15,6 +15,8 @@ data:
   arch_packages:
     aarch64:
       - bpftrace
+	  - opencsd
```

Comment:
This won't work, there needs to be the same indentation (spaces) as for the line above.